bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/MetadataClientDriver.java

```diff
@@ -103,4 +105,28 @@ LedgerManagerFactory getLedgerManagerFactory()
      *            listener listening on metadata client session states.
      */
     void setSessionStateListener(SessionStateListener sessionStateListener);
+
+    /**
+     * Return health check is enable or disable.
+     *
+     * @return true if health check is enable, otherwise false.
+     */
+    default CompletableFuture<Boolean> isHealthCheckEnabled() {
+        return FutureUtils.value(true);
+    }
+
+    /**
+     * Disable health check.
+     */
+    default CompletableFuture<Void> disableHealthCheck() {
+        return FutureUtils.Void();
```

Comment:
I thought  more about this.
Probably it is better to report an error by default in this method for drivers that do not support this feature.
Otherwise people will think that they disabled the health check but they actually didn't

Reply:
like this:
    /**
     * Disable health check.
     */
    default CompletableFuture<Void> disableHealthCheck() {
        new Exception("Not support disableHealthCheck");
    }